docs/tutorials/02_vibrational_structure.ipynb

```diff
@@ -0,0 +1,761 @@
+      "+ 1.175575188382615 * IIIXXIIIIIYY\n",
+      "+ 1.175575188382615 * IIIYYIIIIIYY\n",
+      "- 0.5021409375000001 * IIIIXXIIIIXX\n",
+      "- 0.5021409375000001 * IIIIYYIIIIXX\n",
+      "- 0.5021409375000001 * IIIIXXIIIIYY\n",
+      "- 0.5021409375000001 * IIIIYYIIIIYY\n"
+     ]
+    }
+   ],
+   "source": [
+    "vibrational_problem = VibrationalStructureProblem(driver, num_modals=3, truncation_order=2)\n",
+    "second_q_ops = vibrational_problem.second_q_ops()\n",
+    "\n",
+    "qubit_converter = QubitConverter(mapper = DirectMapper())\n",
+    "\n",
+    "qubit_op = qubit_converter.convert(second_q_ops[0])\n",
+    "\n",
+    "print(qubit_op)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that the Hamiltonian is ready, it can be used in a quantum algorithm to find information about the vibrational structure of the corresponding molecule. Check out our tutorials on Ground State Calculation and Excited States Calculation to learn more about how to do that in Qiskit!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<h3>Version Information</h3><table><tr><th>Qiskit Software</th><th>Version</th></tr><tr><td>Qiskit</td><td>0.24.1</td></tr><tr><td>Terra</td><td>0.17.0.dev0+ad88761</td></tr><tr><td>Aer</td><td>0.7.6</td></tr><tr><td>Ignis</td><td>0.5.2</td></tr><tr><td>Aqua</td><td>None</td></tr><tr><td>IBM Q Provider</td><td>0.12.2</td></tr><tr><th>System information</th></tr><tr><td>Python</td><td>3.7.10 (default, Feb 26 2021, 10:16:00) \n",
+       "[Clang 10.0.0 ]</td></tr><tr><td>OS</td><td>Darwin</td></tr><tr><td>CPUs</td><td>4</td></tr><tr><td>Memory (Gb)</td><td>16.0</td></tr><tr><td colspan='2'>Thu Apr 01 10:23:40 2021 CEST</td></tr></table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div style='width: 100%; background-color:#d5d9e0;padding-left: 10px; padding-bottom: 10px; padding-right: 10px; padding-top: 5px'><h3>This code is a part of Qiskit</h3><p>&copy; Copyright IBM 2017, 2021.</p><p>This code is licensed under the Apache License, Version 2.0. You may<br>obtain a copy of this license in the LICENSE.txt file in the root directory<br> of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.<p>Any modifications or derivative works of this code must retain this<br>copyright notice, and modified files need to carry a notice indicating<br>that they have been altered from the originals.</p></div>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import qiskit.tools.jupyter\n",
+    "%qiskit_version_table\n",
+    "%qiskit_copyright"
```

Comment:
I am going to ask Matthew about this version table that gets added at the end. Its still showing Aqua - which maybe is fine for now, but it does not include the new domains and hence does not show what version of Qiskit Nature this was run with!